### PR TITLE
Add single layer test for ROIPooling

### PR DESCRIPTION
**Pull request by kmolchan**
Add new shared test for ROIPooling layer.

### CODE REVIEW

File: inference-engine/tests/ngraph_functions/src/roi_pooling.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+namespace ngraph {
+namespace builder {
+
+std::shared_ptr<Node> makeROIPooling(const Output<Node>& input,
+                                     const Output<Node>& coords,
+                                     const Shape& output_size,
+                                     const float spatial_scale,
+                                     const ngraph::helpers::ROIPoolingTypes& roi_pool_type) {
+    std::string roi_pool_method = (roi_pool_type == helpers::ROIPoolingTypes::ROI_MAX) ? "max" : "bilinear";
```
Comment (alpkn) [inline inference-engine/tests/ngraph_functions/src/roi_pooling.cpp:19]: one more ternary here with `"undef"` could help to signal about the incomplete code changes.

**Review by alpkn — CHANGES_REQUESTED** [review]

File: inference-engine/tests/ngraph_functions/src/roi_pooling.cpp (hunk 1 above)
Comment (kmolchan) [reply inference-engine/tests/ngraph_functions/src/roi_pooling.cpp]: Add runtime error by analog with Eltwise builder

**Review by alpkn — APPROVED** [review]

**Review by iefode — COMMENTED** [review]
I strongly recommend develop the ngraph reference implementation for this op. Ticket (33236 & 37530)

**Comment by ggalieroc** [comment]
As part of ROIPooling reference implementation #2903 we need to add single layer tests. This PR covers a part of #2903 (single layer test definitions). In order to instantiate tests I would need this PR to be merged. If you do not plan to merge this PR, kindly close it so I can cherry-pick these commits and keep @kmolchan as the original author.

**Comment by iefode** [comment]
Close this PR. @ggalieroc Will enable these tests for CPU and Myriad plugin in [his PR](https://github.com/openvinotoolkit/openvino/pull/2903)

**Pull request closed without merging**